src/NServiceBus.Gateway/Gateway.cs

```diff
@@ -35,7 +35,7 @@ public class Gateway : Feature
     {
         internal Gateway()
         {
-
+            DependsOn("DelayedDelivery");
```

Comment:
What happens when the GW is explicitly enabled the there `DelayedDelivery` feature is disabled? (do we throw a meaningful ex?)